ts/Core/Chart/Chart.ts

```diff
@@ -3724,6 +3724,12 @@ class Chart {
 
 }
 
+addEvent(Chart, 'afterSetChartSize', function (e: { skipAxes: boolean }): void {
+    if (!e.skipAxes) {
+        this.renderer.align();
```

Comment:
Shouldn't we just move this call inside the `setChartSize` function instead?

https://github.com/highcharts/highcharts/blob/4905235f0d4a6ffd93668478b48434fb182621b0/ts/Core/Chart/Chart.ts#L1921-L1924

```ts
        if (!skipAxes) {
            chart.axes.forEach(function (axis: Highcharts.Axis): void {
                axis.setAxisSize();
                axis.setAxisTranslation();
            });            
            renderer.align();
        }
```